js/views/homeVw.js

```diff
@@ -421,8 +439,26 @@ module.exports = Backbone.View.extend({
     this.$el.find('.js-homeSearchItemsClear').addClass('hide');
   },
 
+  setListingsBlockedCount: function(count) {
+    var text = '';
+
+    count = count || 0;
+    if (this.blockedItemCount === count) return;
+    this._blockedItemCount = count;
+    
+    if (count) {
+      text = count + ' blocked item' + (count !== 1 ? 's' : '') + ' not shown';
+    }
+
+    this.$listingsBlockedCount = this.$listingsBlockedCount || this.$('.homeGridItems .js-blocked-listings-count');
```

Comment:
This is an interesting pattern, setting and storing the jQuery object here instead of separately after render.